selftests/unit/test_job.py

```diff
@@ -130,6 +130,25 @@ def post_tests(self):
         self.assertEqual(myjob.unique_id[::-1],
                          open(os.path.join(myjob.logdir, "reversed_id")).read())
 
+    @unittest.skip("Issue described at https://trello.com/c/qgSTIK0Y")
```

Comment:
You're right.  I was reluctant to change that, because it could mean closing the Trello card without solving the **Job** class issue, but we can keep the card anyway.